manifests/oracle-cluster/prod/helium/account-postgres-sink-service.yaml

```diff
@@ -25,6 +25,9 @@ data:
               "table": "mobile_hotspot_infos",
               "schema": "public"
             }
+          ],
+          "crons": [
+            { "schedule": "0 0 * * *", "type": "integrity-check" }
```

Comment:
Given the potential impact, can we change these crons to working hours?

Reply:
@ChewingGlass this should run at midnight right?

Reply:
Yeah, better to run at utc midnight because of what it's correcting